src/UI/ListView.elm

```diff
@@ -299,6 +308,20 @@ withActionBar config (SelectList prop opt) =
     SelectList prop { opt | actionBar = Just config }
 
 
+{-| Adds a button to toggle a custom dropdown element.
+
+    ListView.withCustomDropdown customDropdownView
+        someListView
+
+-}
+withCustomDropdown : msg -> Bool -> Element msg -> ListView object msg -> ListView object msg
```

Comment:
In theory that's a Context Menu, and in this case, one hidden in an "extra button". What do you think of calling "withExtraMenu"?

Reply:
Maybe `withCustomExtraMenu`.

Because we may better define this layout later and add an `withExtraMenu : DropdownMenu msg -> ListView object msg -> ListView object msg`.